Resources/Prototypes/Entities/Objects/Specific/Mining/ore_bag.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  name: ore bag
+  id: OreBag
+  parent: BaseStorageItem
+  description: A robust bag for salvage specialists and miners alike to carry large amounts of ore.
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Objects/Specific/Mining/ore_bag.rsi
+    state: icon
+  - type: Clothing
+    sprite: Objects/Specific/Mining/ore_bag.rsi
+    QuickEquip: false
+    Slots:
+    - belt
+    size: 300
+  - type: Storage
+    capacity: 300
```

Comment:
300 is way too much. ik ore bags in 13 held a shit ton but i'm not really a fan of it except for maybe as a research item in the future. 100-150 is good enough